ui/Hotkeys.svelte

```diff
@@ -11,16 +10,37 @@
   };
 
   const onKeydown = event => {
+    const OSKey = navigator.platform.includes("Mac")
```

Comment:
```suggestion
    const osKey = navigator.platform.includes("Mac")
```

Convention in js/ts/svelte is camelCase for var/let/const names.

Reply:
https://github.com/radicle-dev/radicle-upstream/commit/79c719e7030813f8769a00892116d4b1c31397b3